airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -147,3 +147,13 @@ class Config:
             "A list which contains insights entries, each entry must have a name and can contains fields, breakdowns or action_breakdowns)"
         ),
     )
+
+    insights_lookback_window: Optional[PositiveInt] = Field(
```

Comment:
we should make this `default_insights_lookback_window` and apply it only to the default insights synced by this connector, and separately we should allow configuring this on a per-insight level in the custom insights

Reply:
Hey @vladimir-remar I'm not sure you ended up implementing @sherifnada suggestion. Is anything blocking you from going in the suggested direction?

Reply:
Hello @alafanechere, maybe I'm wrong, I just replicate the same approach in the custom insight part.
https://github.com/airbytehq/airbyte/blob/c6d00d77a47a830f9bda3da6798734828de0e383/airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py#L80